docs/lib/auth/fragments/ios/signin_next_steps/10_signin.md

```diff
@@ -0,0 +1,17 @@
+
+The signin apis if it succeeds you will get back an associated value of type `AuthSignInResult`, and you can access the next step through the property `nextStep` of `AuthSignInResult`. 
```

Comment:
```suggestion
When called successfully, the signin APIs will return an `AuthSignInResult`. Inspect the `nextStep` property in the result to see if additional signin steps required.
```

Reply:
minor grammar fix: "to see if additional signin steps are required"